go/src/koding/klientctl/team.go

```diff
@@ -3,15 +3,63 @@ package main
 import (
 	"encoding/json"
 	"fmt"
-	"koding/klientctl/endpoint/team"
 	"os"
+	"text/tabwriter"
+
+	"koding/kites/kloud/team"
+	epteam "koding/klientctl/endpoint/team"
 
 	"github.com/codegangsta/cli"
 	"github.com/koding/logging"
 )
 
+func TeamList(c *cli.Context, log logging.Logger, _ string) (int, error) {
+	opts := &epteam.ListOptions{
+		Slug: c.String("slug"),
+	}
+
+	teams, err := epteam.List(opts)
+	if err != nil {
+		return 0, err
+	}
+
+	if len(teams) == 0 {
```

Comment:
As you wish, but this is opposite what we have now currently. I tried to return various ErrNotFound errors when things to return were not found.